### DISCUSSION TITLE
Which CIS Benchmark

### DISCUSSION THREAD

**Discussion by sampurcel** [0x6d69636b/windows_hardening#170] [General]
Hi All,

I installed hardening kitty just coming up to a year ago and now I'm arguing with my DevOps engineer over who did a better job hardening each network. His VMs are built from the L2 benchmark and I assume this script will be equivelant to L1, but can someone please let me know which level it is?

**Comment by 0x6d69636b** [comment]
CIS benchmark lists include L1 and L2 recommendations
